critiquebrainz/frontend/external/musicbrainz_db/helpers.py

```diff
@@ -2,18 +2,16 @@
 from sqlalchemy.orm import joinedload
 
 
-def entity_relation_helper(db, target_type, source_type, source_entity_id):
+def entity_relation_helper(db, target_type, source_type, source_entity_ids, includes_data):
```

Comment:
Would a name like `get_relationship_info` be more informative for this function?

Also if this function doesn't return anything, what does it do? Update one of the arguments passed to it? Can you describe this (in the docstring please)?